content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -470,7 +470,7 @@ for scaling down which allows a 100% of the currently running replicas to be rem
 means the scaling target can be scaled down to the minimum allowed replicas.
 For scaling up there is no stabilization window. When the metrics indicate that the target should be
 scaled up the target is scaled up immediately. There are 2 policies where 4 pods or a 100% of the currently
-running replicas will be added every 15 seconds till the HPA reaches its steady state.
+running replicas may at most be added every 15 seconds till the HPA reaches its steady state.
```

Comment:
Even better: rewrite the relevant sentence in the active voice. Eg: “the control plane waits at least 15 seconds between&hellip;”

Reply:
👋 @pbetkier .
Does this edit make sense?

Two policies control the addition of no more than four pods or 100% of the currently
running replicas every 15 seconds until the HPA reaches its steady state.